vendor/github.com/OpenBazaar/go-ethwallet/util/util.go

```diff
@@ -119,3 +122,29 @@ func SigRSV(isig interface{}) ([32]byte, [32]byte, uint8) {
 
 	return R, S, V
 }
+
+// EnsureCorrectPrefix ensures we have 0x prefix
+func EnsureCorrectPrefix(str string) string {
+	if strings.HasPrefix(str, "0x") {
+		return str
+	}
+	return "0x" + str
+}
+
+// CreateChainHash is a wrapper to the chainhash.new hash function
+// this allows for a cleaner way to check if we are not in any way
+// letting the 0x prefix hinder the chainhash generation
+func CreateChainHash(str string) (*chainhash.Hash, error) {
+	hash, err := chainhash.NewHashFromStr(str)
+	if err.Error() == "max hash string length is 64 bytes" {
+		hash, err = chainhash.NewHashFromStr(strings.TrimPrefix(str, "0x"))
+		if err != nil {
+			hash, err = chainhash.NewHashFromStr("")
+			if err != nil {
+				log.Println("err : ", err.Error())
```

Comment:
Can we please use go-logging instead of printing to Stdout? (See https://github.com/OpenBazaar/openbazaar-go/blob/master/repo/init.go#L22)